library/src/commonMain/kotlin/org/jraf/klibnotion/internal/client/NotionService.kt

```diff
@@ -218,17 +217,15 @@ internal class NotionService(private val httpClient: HttpClient) {
 
     // region Search
 
-    suspend fun searchPages(parameters: ApiSearchParameters, startCursor: String?): ApiResultPage<ApiPage> {
+    suspend fun searchPages(parameters: ApiSearchParameters): ApiResultPage<ApiPage> {
         return httpClient.post("$BASE_URL/$SEARCH") {
-            if (startCursor != null) parameter(START_CURSOR, startCursor)
             contentType(ContentType.Application.Json)
             body = parameters
         }
     }
 
-    suspend fun searchDatabases(parameters: ApiSearchParameters, startCursor: String?): ApiResultPage<ApiDatabase> {
+    suspend fun searchDatabases(parameters: ApiSearchParameters): ApiResultPage<ApiDatabase> {
```

Comment:
This has the same implementation as `searchPages`, could it be collapsed to just `search`?
Or at least alias to a single implementation?

Reply:
You are totally right, let's have only one named `search` 👍

Reply:
I'll create a separate PR

Reply:
Whilst the API is the same, the API response is encoded differently here, so we should think about merging the expected API response, especially since the API might respond with either a page or a database, which could cause a deserialisation failure.

Reply:
Ah you're right :) Oh well!